Competition-2020/src/main/include/Robot.h

```diff
@@ -15,9 +15,30 @@
 
 using namespace frc;
 
+class XboxController2 : public XboxController {
+  public:
+    float scale;
+
+    void set_scale(float s) {
+      scale = abs(s);
+    }
+
+    double GetRawAxis(int axis) {
+      if(GenericHID::GetRawAxis(axis) < 0.1 && GenericHID::GetRawAxis(axis) > -0.1) {
+        return 0;
+      }
+      else {
+        return scale * GenericHID::GetRawAxis(axis);
```

Comment:
This should be bounded between the range of -1.0 to 1.0; right now it could go higher or lower